src/interfaces/ITokenHopper.sol

```diff
@@ -45,7 +45,6 @@ interface ITokenHopper {
     struct HopperConfiguration {
         // Initial Funds
         address token;              // Each hopper will hold exactly one token type.
-        uint256 initialAmount;      // The amount to supply the hopper on initialization.
 
         // Behavior
         uint256 cooldownSeconds;    // The number of seconds minimally required between each action.
```

Comment:
set action generator to an interface

Reply:
we just wrap it in one place. this way we minimize interface-to-interface imports, seems fine as-is to me